inst/rmarkdown/templates/josis/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,177 @@
+---
+title: "Long Title for JOSIS Article"
+runningtitle: "Short Title for JOSIS Article"
+output: rticles::josis_article
+# to use bookdown cross references
+# output:
+#   bookdown::pdf_book:
+#     base_format: rticles::josis_article
+author1: Nameof Author1
+affil1: Department, Institution, Country
+author2: Nameof Author2
+affil2: Department, Institution, Country
+keywords: "add, five, to, ten, comma, separated, keywords"
+abstract: "Add your article abstract here,
+  150-250 words."
+bibliography: josisexample.bib
+---
+
+<!-- Created thanks to guidance at -->
+<!-- https://github.com/rstudio/rticles/blob/master/.github/PULL_REQUEST_TEMPLATE.md -->
+
+# Introduction
+
+This template provides a guide to formatting articles for submission to the Journal of Spatial Information Science, JOSIS, \burl{http://josis.org}. When preparing an article for submission, please follow this template closely, referring to past JOSIS articles (open access on the JOSIS web site) for further examples. 
+
+# RMarkdown
+
+The template can handle bullet points:
+
+- a
+- b
+
+And tables (see Table \@ref(tab:t1)):
+
+```{r t1}
+knitr::kable(head(mtcars), caption = "Test", booktabs = TRUE)
+```
+
+You can use concise citation syntax, e.g. [@champion11].
+
+# Author guidelines
+
+
+
+\subsection{Manuscript preparation}
```

Comment:
`\subsection` is `## ` in markdown 

and lower level translate to lower header Level in LaTeX

Maybe it would be better to use Markdown here, don't you think ?